client/src/locales/en-US.json

```diff
@@ -1625,7 +1625,28 @@
                 "description": "You can change your credit card at any time.",
                 "update": "Update"
             },
-            "processing": "Your order is being processed."
+            "processing": {
+                "requestSent": {
+                    "title": "Request sent",
+                    "description": "Your request is being processed. We will contact you within 4 working days."
+                },
+                "estimateLinked": {
+                    "title": "Quotation signature pending",
+                    "description": "Please sign and send back the quotation received by e-mail to get your invoice."
+                },
+                "invoiceToBePaid": {
+                    "title": "Invoice to be paid",
+                    "description": "Awaiting validation for your bank transfer to finalize your order."
+                },
+                "organizationAvailable": {
+                    "title": "Organization available",
+                    "description": "Our team is setting up your organization. You'll receive an email as soon as everything is ready."
```

Comment:
If the invoice is paid, the organization is already existing and will be automatically 'set up', so this sentence does not make really much sense.